src/features/classification/Classification.js

```diff
@@ -24,32 +25,51 @@ type Props = {
     itemName?: string,
     maxAppCount?: number,
     messageStyle?: typeof STYLE_INLINE | typeof STYLE_TOOLTIP,
+    modifiedAt?: string,
+    modifiedBy?: string,
     name?: string,
     onClick?: (event: SyntheticEvent<HTMLButtonElement>) => void,
-};
+} & InjectIntlProvidedProps;
 
 const Classification = ({
     definition,
     className = '',
     controls,
     controlsFormat,
+    intl,
     isLoadingControls,
     maxAppCount,
     messageStyle,
+    modifiedAt,
+    modifiedBy,
     name,
     itemName = '',
     color,
     onClick,
 }: Props) => {
     const isClassified = !!name;
     const hasDefinition = !!definition;
+    const hasModifiedAt = !!modifiedAt;
+    const hasModifiedBy = !!modifiedBy;
     const hasSecurityControls = !!controls;
     const isTooltipMessageEnabled = isClassified && hasDefinition && messageStyle === STYLE_TOOLTIP;
     const isInlineMessageEnabled = isClassified && hasDefinition && messageStyle === STYLE_INLINE;
     const isNotClassifiedMessageVisible = !isClassified && messageStyle === STYLE_INLINE;
     const isControlsIndicatorEnabled = isClassified && isLoadingControls && messageStyle === STYLE_INLINE;
     const isSecurityControlsEnabled =
         isClassified && !isLoadingControls && hasSecurityControls && messageStyle === STYLE_INLINE;
+    const isModifiedMessageVisible = isClassified && hasModifiedAt && hasModifiedBy;
+
+    let formattedModifiedAt;
+
+    if (isModifiedMessageVisible) {
+        const d = new Date(modifiedAt);
```

Comment:
We try to avoid abbreviation, even for simple variable names.

Reply:
Ack. Copied from DateField.js but will make the improvement.